src/pre.js

```diff
@@ -94,13 +95,16 @@ var glpkPromise = new Promise(function (resolve) {
 					housekeeping(P);
 					return FS.readFile(name, { encoding: 'utf8' });
 				},
-				'solve': function (lp, options) {
+				'solve': function (lp, opt) {
 					
-					const settings = _.defaults(options, {
-							msgLev: 1,
-							tmLim: 120000,
-							mipGap: 0
-					})
+					const opt_ = opt || lp.options || {};
+					
+			    const options = {
+			        preSolve: typeof opt_.preSolve !== 'undefined' ? +(!!opt_.preSolve) : 1,
```

Comment:
I think presolve is one word, so maybe no need to `preSolve` and instead `presolve`

Reply:
it WAS `presolve` before, and I reflected that resolve is of course two words and I should change!